From 49b8c274947d4b625c2a9e0a1fdcb631a73d4903 Mon Sep 17 00:00:00 2001
From: Kenneth Graunke <kenneth@whitecape.org>
Date: Mon, 14 Jan 2019 00:32:33 -0800
Subject: [PATCH] NOT-FOR-UPSTREAM-YET: loader - prefer iris over i965

This lets me avoid having to set MESA_LOADER_DRIVER_OVERRIDE everywhere
---
 src/loader/pci_id_driver_map.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/loader/pci_id_driver_map.h b/src/loader/pci_id_driver_map.h
index 99fe4ad2161..9a7f037caec 100644
--- a/src/loader/pci_id_driver_map.h
+++ b/src/loader/pci_id_driver_map.h
@@ -83,8 +83,8 @@ static const struct {
    int (*predicate)(int fd);
 } driver_map[] = {
    { 0x8086, "i915", i915_chip_ids, ARRAY_SIZE(i915_chip_ids) },
-   { 0x8086, "i965", i965_chip_ids, ARRAY_SIZE(i965_chip_ids) },
    { 0x8086, "iris", iris_chip_ids, ARRAY_SIZE(iris_chip_ids) },
+   { 0x8086, "i965", i965_chip_ids, ARRAY_SIZE(i965_chip_ids) },
    { 0x1002, "radeon", r100_chip_ids, ARRAY_SIZE(r100_chip_ids) },
    { 0x1002, "r200", r200_chip_ids, ARRAY_SIZE(r200_chip_ids) },
    { 0x1002, "r300", r300_chip_ids, ARRAY_SIZE(r300_chip_ids) },
-- 
2.18.1

